it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -343,7 +343,6 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
         stubRetrieveCtutr(testJourneyId)(status = OK, body = testCtutr)
         stubValidateIncorporatedEntityDetails(testCompanyNumber, Some(testCtutr))(OK, Json.obj("matched" -> true))
         stubStoreIdentifiersMatch(testJourneyId, identifiersMatch = true)(status = OK)
-        stubCreateBusinessVerificationJourney(testCtutr, testJourneyId)(status = CREATED)
```

Comment:
wonder Why it has been removed ?

Reply:
In the case of a successful match for a limited company the controller should save the match status and redirect control to the Business verification controller. The integration tests checks the value of the redirect. However, the test does not actually perform the redirect. As such "createBusinessVerificationJourney" is not invoked in this test because the method is called from the BusinessVerificationController.